src/foraging_gui/Foraging.py

```diff
@@ -3706,8 +3645,8 @@ def _update_photometery_timer(self,time):
         if len(str(seconds)) == 1:
             seconds = '0{}'.format(seconds)
         if not self.ignore_timer:
-            self.WarningLabelStop.setText('Running photometry baseline: {}:{}'.format(minutes,seconds))
-            self.WarningLabelStop.setStyleSheet(self.default_warning_color)
+            logging.info('Running photometry baseline: {}:{}'.format(minutes,seconds),
```

Comment:
Right now this timer gets updated every second or something. Will that timer still be visible to the user?

Reply:
Yes but it won't be one message it will keep adding a line to the warning box with the time left. Is this okay?

Reply:
That seems like it will clog up the warning box. Can you either delete the previous timer message from the queue? Or maybe we just keep that warning out of the queue?

Reply:
Deleting it from the warning box will be hard. I think best approach would be to add the count down some where else like maybe on the excitation button?

Reply:
that works!

Reply:
Okay I actually changed it to just add a lablel the warning_widget and then update that.

Reply:
![image](https://github.com/user-attachments/assets/45137af2-0f3f-40ff-a726-2d84a5de28c2)

I've changed the text to be the default color in a recent push